avm/src/daemon/linux.rs

```diff
@@ -73,6 +73,7 @@ async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, fact
   let pwd = var("PWD").unwrap();
   let dockerfile = read(format!("{}/Dockerfile", pwd));
   let app_tar_gz = read(format!("{}/app.tar.gz", pwd));
+  let env_file = read(format!("{}/.anycloud.env", pwd));
```

Comment:
per https://github.com/alantech/deploy/pull/54/files#r594423152, we should check that it exists like the other 2 files in the line below